### ISSUE TITLE
max-attributes-per-line

### ISSUE THREAD

**Issue by cybermerlin** [vuejs/eslint-plugin-vue#1331]
hi. sorry about this task.

for rule:
```
    'vue/max-attributes-per-line': ['error', {
      'singleline': 3,
      'multiline': {
        'max': 1,
        'allowFirstLine': true
      }
    }],
```

for this case:
```
        <q-btn v-if="!!attributesCast.length"
dense flat icon="settings" round
@click="toggleFilter">
```

and the same situation on the vue docs page https://eslint.vuejs.org/rules/max-attributes-per-line.html, when u press on the button Preview and change input data on left side
![image](https://user-images.githubusercontent.com/1460479/95738179-7c4acc00-0c91-11eb-8ea9-26bdf2fb38b4.png)

-------------

**Checklist**

- [ ] I have tried restarting my IDE and the issue persists.
- [ ] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** 
- **eslint-plugin-vue version:** 
- **Node version:** 
- **Operating System:** 

**Please show your full configuration:**

```json5

```

**What did you do?**	

```vue

```

**What did you expect to happen?**

**What actually happened?**

**Repository to reproduce this issue**

**Comment by ota-meshi** [comment]
Hi @cybermerlin.

You should use the html-indent rule together.
https://eslint.vuejs.org/rules/html-indent.html

Or apply a formatter such as Prettier after auto-fixing with ESLint.

**Comment by cybermerlin** [comment]
I use both. 
But what about this mistake on the Vue.docs page?   (look at the picture)

**Comment by ota-meshi** [comment]
Each rule keeps it simple and doesn't interfere with other style rules. Use a combination of rules that keep the style you want.

**Comment by cybermerlin** [comment]
> Each rule keeps it simple and doesn't interfere with other style rules. Use a combination of rules that keep the style you want.

that's very difficult. 400 line in my .eslintrc. I try to reproduce the Stroustrup style, but combination the prettier with the eslint - like a mastodon reproduce.
